docs/phoenix_guidelines.md

```diff
@@ -0,0 +1,41 @@
+# Best Practices for Interacting with Phoenix
+
+Phoenix is a chemistry-focused discovery agent. To get the most out of each Phoenix run, follow these guidelines for formulating effective queries:
+
+## 1. Be specific about molecular inputs.
+When asking about molecules, provide SMILES strings, CAS numbers, IUPAC names, or molecular formulas explicitly. Phoenix can work with multiple representation types, but being explicit helps ensure accurate results. A chemist familiar with your field should be able to unambiguously identify the molecule(s) you're asking about.
+
+## 2. Specify desired outputs clearly.
+Clearly state what you need from Phoenix. It can be a synthesis route, molecular property prediction, safety assessment, literature-backed answer, or a combination of these. The more specific you are about the outputs you need, the better Phoenix can select appropriate tools and provide actionable results.
+
+## 3. Use proper chemical terminology and notation.
+Employ standard chemical nomenclature and notation. For example, use SMILES representation for reactions: `reactants>reagents>products`. Use standard property names (e.g., ADMET properties like HIA_Hou, BBB_Martins, AMES) when requesting specific molecular properties. This helps Phoenix understand your intent and select the most appropriate computational tools.
+
+## 4. Break down complex queries.
+For multi-part questions, structure them logically so Phoenix can create an effective execution plan. While Phoenix can handle multi-step queries and will plan longer workflows, clearly organizing your query helps ensure all components are addressed systematically.
+
+## 5. Provide context when relevant.
+Include background information about your use case (e.g., "for drug development" or "for a research synthesis") to help Phoenix select appropriate tools and safety considerations. Context about your constraints, goals, or specific requirements enables Phoenix to provide more targeted and useful responses.
+
+## 6. Request specific properties or analyses.
+Instead of asking vaguely about a molecule, specify what you need. For example, request specific ADMET properties, synthetic accessibility scores, solubility predictions, or toxicity data. This allows Phoenix to use the most appropriate computational tools and provide quantitative, actionable results.
```

Comment:
I understand the point of including this, but it seems redundant - all of htis is listed above. maybe put those as bullets on this one or remove this